NameSorterAlpha/NameSorterAlpha/Program.cs

```diff
@@ -22,11 +22,12 @@ public static void Main(string[] args)
                 }
                 catch (MissingInfoException)
                 {
-                    Console.WriteLine("The person in line " + line_count + " is missing information");
+                    Console.WriteLine("The person in line " + line_count + " is missing information.\n");
```

Comment:
Displaying the name or entry would be more useful than showing the line of the input where it misses an info. This helps for larger input set.